docs/ruby/knapsack/readme.md

```diff
@@ -3,7 +3,6 @@ layout: docs
 title: Knapsack gem README
 permalink: /ruby/knapsack
 ---
-{% include knapsack_gem_utm_cookies.html %}
```

Comment:
Do you want to remove this tracking on purpose?
We were tracking if someone visited knapsack readme and set UTM tag then.

The conversion is visible here:
https://knapsackpro.com/dashboard/admin_statistics/utm_tags?end_date=2022-04-19&start_date=1922-04-19
![image](https://user-images.githubusercontent.com/662545/164051512-85c5a79c-17bc-4bb3-b97a-cef0dfd503dd.png)